agent/connect/ca/plugin/provider.proto

```diff
@@ -49,6 +51,10 @@ message CrossSignCARequest {
     bytes crt = 1;
 }
 
+message StateResponse {
+    bytes state = 1; // JSON-encoded map[string]string
```

Comment:
Do we enforce that its JSON-encoded. Given the PR title this seems like its intended for opaque data that is encoded somehow.